mods/cnc/rules/tech.yaml

```
@@ -13,10 +13,10 @@ V19:
 	Tooltip:
 		Name: actor-v19-name
 	TooltipDescription@ally:
-		Description: Provides additional funds.
+		Description: actor-oilb-desc1
```

```suggestion
		Description: actor-oilb.ally-desc
```

It would be better if translations were structure like this

---

fixed